textract/parsers/__init__.py

```diff
@@ -19,9 +19,10 @@
 # here so the default is used on both the process function and also by
 # the command line interface
 DEFAULT_ENCODING = 'utf_8'
+DEFAULT_EXTENSION = None
 
 
-def process(filename, encoding=DEFAULT_ENCODING, **kwargs):
+def process(filename, encoding=DEFAULT_ENCODING, extension=DEFAULT_EXTENSION, **kwargs):
```

Comment:
@deanmalmgren , I will remove extension=DEFAULT_EXTENSION argument, and I will extract the extension argument from kwargs. I think that is better, right?